doc/source/locale/zh_CN/LC_MESSAGES/development/contributing_codebase.po

```diff
@@ -45,16 +48,21 @@ msgid ""
 "result, that is, we need it to be as *backwards compatible* as possible "
 "to avoid mass breakages."
 msgstr ""
+"此外，重要的是，我们不要对代码进行突然更改"
+"这可能会导致大量用户代码中断"
+"也就是说,我们需要它尽可能向后兼容，以避免大规模中断。"
```

Comment:
**Please use Chinese punctuations in your translation.**

This should be applied consistently throughout the codebase, including other parts of this pull request.


```suggestion
"此外，重要的是，我们不要对代码进行突然的不兼容更改"
"这可能会导致大量用户代码发生错误"
"也就是说，我们需要它尽可能向后兼容，以避免大规模兼容性问题。"
```